### PR TITLE
chore: bump cw-dex-astroport version

### PR DESCRIPTION
**Pull request by ApolloGie**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # [0.3.1] - 2024-02-06
 
+- Replace `cw-dex/astroport` with `cw-dex-astroport`.
```
Comment (apollo-sturdy) [inline CHANGELOG.md:10]: Incorrect version and date

File: Cargo.toml [hunk 2]
```diff
@@ -52,6 +52,7 @@ cosmwasm-schema = "1.2.1"
 apollo-cw-asset = "0.1.0"
 cw-controllers = "1.0.1"
 cw-dex = "0.5.1"
+cw-dex-astroport  = { version = "0.2.0-rc.1", optional = true}
```
Comment (apollo-sturdy) [inline Cargo.toml:55]: You'll also have to stop using Pool enum from `cw-dex` on https://github.com/apollodao/cw-dex-router/blob/master/src/operations.rs#L7

and create your own pool enum with variants based on feature flags in this crate.

**Review by apollo-sturdy — CHANGES_REQUESTED** [review]

File: Makefile.toml [hunk 3]
```diff
@@ -9,8 +9,8 @@ skip_core_tasks = true
 RUST_OPTIMIZER_VERSION = "0.15.0"
 # Use rust version from rust-optimizer Dockerfile (see https://github.com/CosmWasm/rust-optimizer/blob/main/Dockerfile#L1)
 # to be sure that we compile / test against the same version
-RUST_VERSION = "1.72.0"
-NIGHTLY_VERSION = "nightly-2024-02-12"
+RUST_VERSION = "1.75.0"
```
Comment (pacmanifold) [inline Makefile.toml:12]: lets run 1.79.0. for some reason 1.75 takes 30 min for me to compile the latest osmosis-std (after they upgraded go version to 1.23)

Comment (pacmanifold) [reply Makefile.toml:12]:
```suggestion
RUST_VERSION = "1.79.0"
```

File: Makefile.toml [hunk 4]
```diff
@@ -9,8 +9,8 @@ skip_core_tasks = true
...
+NIGHTLY_VERSION = "nightly-2023-08-29"
```
Comment (pacmanifold) [inline Makefile.toml:13]: why downgrade to older nightly?

File: Cargo.toml [hunk 5]
```diff
@@ -51,17 +51,17 @@ cosmwasm-std = "1.5.4"
 cosmwasm-schema = "1.2.1"
 apollo-cw-asset = "0.1.0"
 cw-controllers = "1.0.1"
-cw-dex = "0.5.3"
+cw-dex = "0.6.0-rc.5"
```
Comment (pacmanifold) [inline Cargo.toml]: remove rc

**Review by pacmanifold — CHANGES_REQUESTED** [review]

**Review by pacmanifold — APPROVED** [review]

**Pull request merged**